src/java/picard/analysis/CollectWgsMetricsFromQuerySorted.java

```diff
@@ -270,6 +280,21 @@ private boolean isNoiseRead(final SAMRecord record) {
     }
 
     /**
+     * @param readPair the read pair
+     * @param minimumMappingQuality the minimum mapping quality
+     * @return true if the read pair is considered chimeric, false otherwise
+     */
+    private boolean isChimericReadPair(final QuerySortedReadPairIteratorUtil.ReadPair readPair, final int minimumMappingQuality) {
```

Comment:
Since @nh13 just did something very similar in CollectAlignmentSummaryMetrics it would probably be good to extract this into a re-usable spot so that there's a reasonable fixed definition of Chimeras somewhere.  I could imagine:
```java
public class ChimeraUtil {
    public static int DEFAULT_INSERT_SIZE_LIMIT = 100000;
    public static int DEFAULT_MAPQ_LIMIT = 20;
    public Set<PairOrientation> DEFAULT_EXPECTED_ORIENTATIONS = EnumSet.of(PairOrientation.FR);

    public void isChimeric(final SAMRecord rec);
    public void isChimeric(final SAMRecord r1, final SAMRecord r2);
```
}